temporal-sdk/src/main/java/io/temporal/common/interceptors/WorkflowClientCallsInterceptor.java

```diff
@@ -233,21 +234,20 @@ public WorkflowStartOutput getWorkflowStartOutput() {
 
   final class WorkflowUpdateWithStartInput<R> {
     private final WorkflowStartInput workflowStartInput;
-    private final UpdateWithStartWorkflowOperation<R> updateOperation;
+    private final StartUpdateInput<R> workflowUpdateInput;
```

Comment:
@cretz pointed out in the Go SDK PR interceptors shouldn't reuse input/output from other interceptors, which I agree with , but the Java SDK has done that historically. I don't think we need to block the PR on this , but we should resist this before we make UwS stable in the Java SDK.